### PR TITLE
Union the translations for "简体中文" and "繁体中文".

### PR DESCRIPTION
**Pull request by ghost**
Union the translations for "简体中文" and "繁体中文".

### CODE REVIEW

**Comment by ghost** [comment]
Ping @nodejs/nodejs-cn, @nodejs/nodejs-tw

**Review by dca — DISMISSED** [review]

File: locale/zh-cn/site.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "author": "Node.js",
   "url": "https://nodejs.org/zh-cn/",
   "locale": "zh-cn",
-  "language": "中文(简体)",
+  "language": "简体中文",
   "languageEnglishVersion": "Chinese Simplified",
```
Comment (wazho) [inline locale/zh-cn/site.json]: https://en.wikipedia.org/wiki/Traditional_Chinese_characters

`Simplified Chinese`

Screen Shot 2021-03-29 at 10 29 32 AM

File: locale/zh-tw/site.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   "author": "Node.js",
   "url": "https://nodejs.org/zh-tw/",
   "locale": "zh-tw",
-  "language": "中文 (繁體)",
-  "languageEnglishVersion": "Traditional Chinese",
+  "language": "繁體中文",
+  "languageEnglishVersion": "Chinese Traditional",
```
Comment (wazho) [inline locale/zh-tw/site.json]: https://en.wikipedia.org/wiki/Traditional_Chinese_characters

`Traditional Chinese`

Screen Shot 2021-03-29 at 10 29 43 AM

**Review by willin — APPROVED** [review]
LGTM

File: locale/zh-tw/site.json (hunk 2 above)
Comment (willin) [reply locale/zh-tw/site.json]: but in our english classes, we never use like this

**Comment by ghost** [comment]
After looking for Oxford Dictionary online, it seems "Chinese Simplified" should be "Chinese (Simplified)", and the correct should be "Simplified Chinese" fitting for "简体中文". So is for the "繁体中文". 

Thanks for @dca and @willin

**Comment by MaledongGit** [comment]
After looking for Oxford Dictionary online, it seems "Chinese Simplified" should be "Chinese (Simplified)", and the correct should be "Simplified Chinese" fitting for "简体中文". So is for the "繁体中文". 

Thanks for @dca and @willin

**Pull request merged**